src/style.css

```diff
@@ -0,0 +1,135 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  line-height: 18px;
+}
+
+body,
+p,
+h2,
+img,
+a {
+  margin: 0;
+}
+
+.card {
+  background: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  padding: 16px;
+}
+
+.card__screen {
+  height: 134px;
+  width: 160px;
+  object-fit: cover;
+  margin: 16px 3px 36px;
+}
+
+.card__title {
+  display: flex;
+  justify-content: center;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  padding-bottom: 16px;
+}
+
+.stars-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  display: inline-flex;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars-container__review {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.stars__star--active {
+  background-image: url(images/star-active.svg);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 16px;
+}
+
+.card__title-price {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+  padding-right: 3px;
+}
+
+.card__number-price {
+  align-self: flex-end;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  padding: 0;
+  border: none;
+  width: 100%;
+}
+
+.card__linkButton {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: #00acdc;
+  border-radius: 5px;
+  text-align: center;
+  border: none;
+  
+  padding: 12px 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
you dont need to repeat the font, you can inherit it

Reply:
not fixed